(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'Rows' object has no attribute 'id'"
p3
sS'layer'
p4
S'C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py'
p5
sS'code'
p6
S'# -*- coding: utf-8 -*-\n# probeer zoiets als\n\n\ndef index():\n    rows = db(db.sensor.id > 0).select()\n    rows = db(db.sensor.naam == "sensor1").select()\n    return dict(message="hello from overzicht.py", rijen=rows.id)\n\nresponse._vars=response._caller(index)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\gluon\\restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py", line 10, in <module>\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\gluon\\globals.py", line 417, in <lambda>\n    self._caller = lambda f: f()\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py", line 8, in index\n    return dict(message="hello from overzicht.py", rijen=rows.id)\nAttributeError: \'Rows\' object has no attribute \'id\'\n'
p11
s.